lib/matplotlib/colorbar.py

```diff
@@ -1213,17 +1225,52 @@ def make_axes_gridspec(parent, **kw):
     pad_s = (1. - shrink) * 0.5
     wh_ratios = [pad_s, shrink, pad_s]
 
+    # make parents a 1-d ndarray if its not already...
+    parents = np.atleast_1d(parents).ravel()
+    # For the subplotspec that these axes belong to, loop through and get
+    # the maximum and minimum index into the subplotspec.  The result is the
+    # size the new gridspec that we will create must be.
+    gsp0 = parents[0].get_subplotspec().get_gridspec()
+    minind = 10000
```

Comment:
Probably better to use `np.inf` and `-np.inf` or use the values from the first parent.